epochconverter/convert.ts

```diff
@@ -0,0 +1,15 @@
+import {element, by, ElementFinder} from 'protractor';
+
+export class YM {
+	public convert_year: ElementFinder = element(by.id("br-d1")).element(by.className("dateform year"));
+	public convert_month: ElementFinder = element(by.id("br-d2")).element(by.className("dateform month"));
+	public convert_button: ElementFinder = element(by.buttonText("Convert"))
+
+	constructor(year:string, month:string){
```

Comment:
Keine Logik im Constructor -> Auslagern in eigene Funktion

Reply:
Ist entfernt worden